descriptors/ModuleDescriptor-template.json

```diff
@@ -33,6 +33,17 @@
         }
       ]
     },
+    {
+      "id": "item-storage-batch-sync",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/item-storage/batch/synchronous",
```

Comment:
It appears the paths have only been updated in the module descriptor and not in the RAML or the tests, is that intentional?

Reply:
Thanks for finding this discrepancy, fixed.